pyuvsim/simsetup.py

```diff
@@ -258,18 +269,20 @@ def read_text_catalog(catalog_csv, input_uv=None, source_select_kwds={}, return_
 
     Args:
         catalog_csv: tab separated value file with the following expected columns:
+            For now, all sources are flat spectrum.
             |  Source_ID: source name as a string of maximum 10 characters
             |  ra_j2000: right ascension at J2000 epoch, in decimal degrees
             |  dec_j2000: declination at J2000 epoch, in decimal degrees
             |  flux_density_I: Stokes I flux density in Janskys
             |  frequency: reference frequency (for future spectral indexing) [Hz]
-                For now, all sources are flat spectrum.
         input_uv: The UVData object for the simulation (needed for horizon cuts)
         source_select_kwds: Dictionary of keywords for source selection.
             Valid options:
             |  lst_array: For coarse RA horizon cuts, lsts used in the simulation [radians]
-            |  latitude_deg: Latitude of telescope in degrees. Used for declination coarse horizon cut.
-            |  horizon_buffer: Angle (float, in radians) of buffer for coarse horizon cut. Default is about 10 minutes of sky rotation.
+            |  latitude_deg: Latitude of telescope in degrees. Used for declination coarse
+            horizon cut.
+            |  horizon_buffer: Angle (float, in radians) of buffer for coarse horizon cut.
+            Default is about 10 minutes of sky rotation.
```

Comment:
This docstring doesn't render properly on RTD: https://pyuvsim.readthedocs.io/en/code-format/classes.html#pyuvsim.simsetup.read_text_catalog